### PR TITLE
[NL] add ability to click on full query history

### PR DESCRIPTION
**Pull request by beets**
will trigger the full stream of queries (on auto=true) if there was more than 1 query in the history

### CODE REVIEW

File: static/js/apps/nl_interface/query_history.tsx [hunk 1]
```diff
@@ -41,14 +41,16 @@ export function QueryHistory(props: QueryHistoryProps): JSX.Element {
...
           <h3>Or try one of these recent queries:</h3>
-          {history.map((query, i) => {
+          {history.map((queries, i) => {
             return (
               <div
                 className="history-item"
                 key={i}
-                onClick={() => props.onItemClick(query)}
+                onClick={() => props.onItemClick(history[i])}
+                title={queries.join(", ")}
```
Comment (pradh) [inline static/js/apps/nl_interface/query_history.tsx:50]: Mostly out of curiosity, where the `;` join happening?

**Review by pradh — APPROVED** [review]
Thanks!

File: static/js/apps/nl_interface/query_history.tsx (hunk 1 above)
Comment (beets) [reply static/js/apps/nl_interface/query_history.tsx:50]: this is just for display purposes. so a long hover over the text will show you the full query (with comma separation).

`onItemClick(history[i])` sends the full array of queries for that "session" to the parent which does the query-by-query handling.

Comment (pradh) [reply static/js/apps/nl_interface/query_history.tsx:50]: thanks!

**Pull request merged**